src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -476,12 +470,8 @@ public String getMockGrpcServiceImplName(Interface service) {
     return className(Name.upperCamel("Mock", service.getSimpleName(), "Impl"));
   }
 
-  /** The method name of getter function call for the given name */
-  public String getGetFunctionCallName(Name name, TypeRef type) {
-    if (type.isRepeated() && !type.isMap()) {
-      return methodName(Name.from("get").join(name).join("list"));
-    } else {
-      return methodName(Name.from("get").join(name));
-    }
+  /** The method call string which combines the given the callee and the called method name */
+  public String methodCall(String callee, String method, List<String> params) {
+    return callee + "." + method + "(" + Joiner.on(", ").join(params) + ")";
```

Comment:
This should not be here, it is language-specific. Only names should be rendered here, and syntax should be rendered in the template.